.github/workflows/bash.yml

```
@@ -0,0 +1,41 @@
+name: Create and publish the Cylc bash Docker image for testing
+
+on:
+  workflow_dispatch:
+
+env:
+  REGISTRY: ghcr.io
+  # IMAGE_NAME: ${{ github.repository }}
+  IMAGE_NAME: cylc/cylc-bash-testing
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
```

Is there a reason not to use the evergreen tags for these Docker actions?
```suggestion
        uses: docker/login-action@v1
```

---

No, it was just a copy paste from gh docs

---

Ah, just noticed you mentioned that already!